bacon/controller/AddTransactionViewController.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  AddTransactionViewController.swift
+//  bacon
+//
+//  Created by Lizhi Zhang on 21/3/19.
+//  Copyright © 2019 nus.CS3217. All rights reserved.
+//
+
+import UIKit
+
+class AddTransactionViewController: UIViewController {
+    
+    var isExpenditure = true
```

Comment:
1) Make it private?
2) Maybe create a config.swift file and refactor this there. This seems to me like some sort of configuration: "when adding a transaction, the default is to record an expenditure", which makes sense (and I agree with) but I think it should belong in a config file so we can tweak all the config options in 1 file instead of looking for hardcoded values across different files.